doc/repoquery.rst

```diff
@@ -64,7 +64,8 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
     Limit the resulting set only to package that owns ``<file>``.
 
 ``--installed``
-    Limit the resulting set to installed packges.
+    Limit the resulting set to installed packges. If in dnf.conf file exclude
+    option is used, the option \-\ `-disableexcludes\ <https://dnf.readthedocs.org/en/latest/command_ref.html#options-label>`_ ensure that all installed packages will be listed.
```

Comment:
please, document it in `command_ref.rst` for `dnf list installed` (with references to disableexcludes) in dnf too.